steps/app-ray-mobile-security/1.3.9/step.yml

```diff
@@ -0,0 +1,83 @@
+title: App-Ray Mobile Security
+summary: |
+  Mobile application security testing with App-Ray's engine via API
+description: |
+  Fully automated mobile application security testing identifies vulnerabilities, mitigate risks, comply with regulations whether you are developer, CISO, CEO.
+website: https://app-ray.co/
+source_code_url: https://github.com/App-Ray/bitrise-step-app-ray-mobile-security
+support_url: https://github.com/App-Ray/bitrise-step-app-ray-mobile-security/issues
+published_at: 2021-01-11T19:17:36.637764711+01:00
+source:
+  git: https://github.com/App-Ray/bitrise-step-app-ray-mobile-security.git
+  commit: dd94f3e7af0b91ebdd0f12a8cc594449729fa1cd
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- test
+toolkit:
+  bash:
+    entry_file: step.sh
+deps:
+  brew:
+  - name: curl
+  apt_get:
+  - name: curl
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- base_url: null
+  opts:
+    description: |-
+      Your App-Ray base URL
+      Example https://demo.app-ray.co/
+      URL MUST end with /
+    is_required: true
+    summary: "Your App-Ray base URL \nExample https://demo.app-ray.co/"
+    title: AppRay instance base URL
+- app_path: null
+  opts:
+    description: Path to the app file you want to upload, generated by Bitrise, usually
```

Comment:
Is AAB supported?
I think it is worth confirming or denying.

Reply:
It is supported

Reply:
Added it to title and escription